Extending/Content-Apps/index-v8.md

```diff
@@ -112,6 +112,8 @@ In a similar way you can limit your content app according to user roles (groups)
         "+role/admin"  // show for 'admin' user group
     ]
 
+If a role restriction is given in the manifest, it overrides any other restrictions on based on type.
```

Comment:
```suggestion
If a role restriction is given in the manifest, it overrides any other restrictions based on type.
```

Reply:
@crgrieve any chance you can accept the suggestion?

Reply:
Done!